als-suggestions/shared/src/main/scala/org/mulesoft/als/suggestions/styler/JsonSuggestionStyler.scala

```diff
@@ -23,6 +28,15 @@ case class JsonSuggestionStyler(override val params: SyamlStylerParams) extends
   override protected def renderYPart(part: YPart, indentation: Option[Int] = None): String =
     JsonRender.render(part, indentation = indentation.getOrElse(0), buildRenderOptions)
 
+  /**
+    * This method is used to escape the backslash character for non snippets.
+    * @param fixedJson rendered and fixed suggestion
+    * @param newText the original suggestion
+    * @return the suggestion escaped
+    */
+  private def escapeNonSnippets(newtext: String): String =
```

Comment:
naming a bit confusing, `newtext` is `fixedJson` and not `raw.newText` (also, scaladocs are no longer relevant)

Reply:
Done